pkg/kubelet/kubelet_getters.go

```diff
@@ -274,6 +274,26 @@ func (kl *Kubelet) getPodVolumePathListFromDisk(podUID types.UID) ([]string, err
 	return volumes, nil
 }
 
+func (kl *Kubelet) getMountedVolumePathListFromDisk(podUID types.UID) ([]string, error) {
+	mountedVolumes := []string{}
+	volumePaths, err := kl.getPodVolumePathListFromDisk(podUID)
+	if err != nil {
+		return volumePaths, err
+	}
+	for _, volumePath := range volumePaths {
+		// Check whether volume path is mounted or not. If one path has error to check, continue to the next check.
```

Comment:
I think the comment should be about *why* it's safe to ignore errors and continue. The opposite is to fail and return error immediately.

Reply:
This function is called by podVolumesExist() which will return true/false to indicate whether pod can be deleted or not. In case of checking error, I am thinking maybe it is better to return error here, and then podVolumeExist() should return true since we are not sure whether paths exist and are mounted. Then the behavior would be if we fail to check volume paths, pods cannot be deleted.

Reply:
Changing this sgtm. Please make the change and either add comment for `podVolumesExist`, or let it return both `bool` and `error` instead.

Reply:
changed the code as discussed. PTAL